web_group_expand/static/src/js/web_group_expand.js

```diff
@@ -0,0 +1,43 @@
+openerp.web_group_expand = function(openerp) {
```

Comment:
use strict; is recommended for all javascript files (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#javascript)
https://github.com/odoo/odoo/blob/8.0/addons/web/static/src/js/openerpframework.js#L35